app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
Please uncomment the `:confirmable` so that The application asks for confirmation of email 👍🏻

Reply:
Great, done, thanks!